wit/ephemeral/spin-config.wit

```diff
@@ -0,0 +1,10 @@
+// Get a configuration value for the current component.
+// The config key must match one defined in in the component manifest.
+get-config: function(key: string) -> expected<string, error>
```

Comment:
As we want to add values other than strings, do we want to have a `get-config-string` function instead?